aim/sdk/objects/plugins/hub_dataset.py

```diff
@@ -4,6 +4,11 @@
 
 @CustomObject.alias('hub.dataset')
 class HubDataset(CustomObject):
+    from aim.utils.deprecation import deprecation_warning
+    deprecation_warning(remove_version='3.17', msg='Using HubDataset is deprecated!\
```

Comment:
Is this really a deprecation? What if one continues to use the older version of deeplake?

Reply:
That‘s a valid question. The name change happened only roughly half a year ago. But seemingly without downwards compatibility nor official deprecation warning. I should have split the deprecation from the deeplake implementation. Also, since both implementations work in parallel.